### DISCUSSION TITLE
I just enabled multidiffusion but it does not appear anywhere in the UI

### DISCUSSION THREAD

**Discussion by ilikenwf** [vladmandic/sdnext#534] [General]
I have tried this with both the default theme and the default gradio theme.

![image](https://user-images.githubusercontent.com/414778/234692477-cc95c3c8-1270-4d43-b881-f332a6703c0f.png)

**Comment by ilikenwf** [comment]
No console errors, there is a browser console error when I select hires fix, but that shouldn't be related. It should appear around the "randomize" accordion for seed travel, shouldn't it?

**Comment by ilikenwf** [comment]
Svelte is throwing a console error when I check/uncheck the high res fix box...and I do see plenty of hidden elements, but those I think are controlled by UI events, not settings?

Literally no extensions are disabled. I'll try disabling and reenabling all.

**Comment by ilikenwf** [comment]
Nope, that didn't do it either. I don't even see the html element it should generate hidden anywhere with the inspector, @vladmandic

**Comment by vladmandic** [comment]
first check if multidiffusion is listed in enabled extensions during startup - it will say in console log.
second, double check with default theme, just to rule out artifacts.

i don't even see controlnet in your screenshot?!

this is how left column looks by default:
![image](https://user-images.githubusercontent.com/57876960/234696014-8119c67c-23d3-4b80-91a8-8cbed85a724b.png)

btw, did you install tomed extension? token merging is already built-in here, you'll see it in settings. no need for extension.

**Comment by ilikenwf** [comment]
I did install that one, will remove...I also had to disable ddetailer for now because my current setup is using shared python libraries and while all my other packages are compatible, mmcv and mmdet don't get along well. I'm running a non-docker "hard mode" install. Let me resync the submodules.

**Comment by ilikenwf** [comment]
Looks like I missed a step when I installed. I have some custom code adjustments, and am using some upstream versions of things here and there, but I think I was just lacking the submodules and then the setup that comes after - I had some of them but believe I forgot to sync them.

Thank you/sorry!

```

 Extensions enabled: ['sd-webui-model-converter', 'sd-extension-aesthetic-scorer', 'sd-extension-system-info',            
                         'sd_theme_editor', 'seed_travel', 'stable-diffusion-webui-rembg', 'sd-webui-controlnet', 'prompt-bracket-checker',       
                         'stable-diffusion-webui-images-browser', 'sd-dynamic-thresholding', 'clip-interrogator-ext', 'a1111-sd-webui-lycoris',   
                         'SwinIR', 'sd-extension-steps-animation', 'ScuNET', 'Lora', 'LDSR', 'multidiffusion-upscaler-for-automatic1111']         
15:56:09-762141 INFO     Extensions enabled: ['stable-diffusion-webui-tokenizer', 'a1111-sd-webui-locon', 'seed_travel',                          
                         'sd-dynamic-thresholding', 'shift-attention', 'stable-diffusion-webui-randomize', 'sd-webui-ar',                         
                         'multidiffusion-upscaler-for-automatic1111']    
```
